src/import-tool.cc

```diff
@@ -445,22 +471,27 @@ int run_command3_and_wait(string command, string arg1, string arg2, string arg3)
     return child_return_code;
 }
 
-int report_importer_log(string importer, vector<string> & extra_args) {
+// this function assumes importer arg is valid
+int report_importer_log(string importer, string log_filename, vector<string> & extra_args) {
     int linecount = 1000;
     if (extra_args.size() == 1) {
         stringstream iss(extra_args[0]);
         iss >> linecount;
     }
+    if (file_exists(log_filename)) {
+        cout << "Printing the last " << linecount << " lines from file " << log_filename << " :" << endl;
+        stringstream oss;
+        oss << linecount;
+        run_command3_and_wait("/usr/bin/tail", "-n", oss.str(), log_filename);
+    } else {
+        cout << "File " << log_filename << " could not be found." << endl;
+    }
+    return 0;
+}
+
+int report_importer_log(string importer, vector<string> & extra_args) {
     if (importer == "triage") {
-        if (file_exists(TRIAGE_IMPORT_LOG_FILENAME)) {
-            cout << "Printing the last " << linecount << " lines from file " << TRIAGE_IMPORT_LOG_FILENAME << " :" << endl;
-            stringstream oss;
-            oss << linecount;
-            run_command3_and_wait("/usr/bin/tail", "-n", oss.str(), TRIAGE_IMPORT_LOG_FILENAME);
-        } else {
-            cout << "File " << TRIAGE_IMPORT_LOG_FILENAME << " could not be found." << endl;
-        }
-        return 0;
+        return report_importer_log(importer, TRIAGE_IMPORT_LOG_FILENAME, extra_args);
```

Comment:
is this meant to have a second conditional for hgnc?

Reply:
added